### PR TITLE
clarify that no quantization config needs to be specified for fp8 405B

### PR DESCRIPTION
**Pull request by ianporada**
* fix typo
* add clarification regarding quantization config
* torch_dtype is already specified in the model config.json so we can just use "auto"

### CODE REVIEW

**Comment by osanseviero** [comment]
Hi @ariG23498 could you take a look at this one please?

**Review by SunMarc — APPROVED** [review]
Thanks @ianporada ! Left a suggestion

File: fp8-405B.ipynb [hunk 1]
```diff
@@ -86,7 +86,7 @@
     "model_name = \"meta-llama/Meta-Llama-3.1-405B-Instruct-FP8\"\n",
     "\n",
     "quantized_model = AutoModelForCausalLM.from_pretrained(\n",
-    "\tmodel_name, device_map=\"auto\", torch_dtype=torch.bfloat16)"
+    "\tmodel_name, device_map=\"auto\", torch_dtype=\"auto\")"
```
[the remark this answers is not in the document]
Comment (ianporada) [reply fp8-405B.ipynb]: Sure makes sense! Removed this change

**Review by SunMarc — APPROVED** [review]
Thanks for iterating ! Can you fix the merge conflits so that I can merge it ?

**Comment by ianporada** [comment]
@SunMarc fixed merge conflicts

**Comment by SunMarc** [comment]
Nice thanks again for the contribution !

**Pull request merged**